### DISCUSSION TITLE
Using Sunshine/Moonlight on Steam Deck with Playnite

### DISCUSSION THREAD

**Discussion by OUSooner911** [LizardByte/community#125] [Sunshine]
Hey everyone! I'm super new to github and even newer to this but after hours of playing with settings, I discovered how to utilize Playnite with the Steam Deck and even figured out how to mimic the ever so popular "Gamestream_Launchpad" exe for Playnite

To do so, I first had to setup the Sunshine program on my PC and this youtube video came in handy: https://www.youtube.com/watch?v=Wb8j8Ojd4YQ

Once complete, I added a new Application and called it Playnite. I then performed the following actions:

- Command Preparations (you'll need the QRes app which can be downloaded at https://www.softpedia.com/get/Multimedia/Video/Other-VIDEO-Tools/QRes.shtml   . Then, I moved the exe over to my System32 folder.)
> 1.  Set "Do" to:    qres.exe /x:1280 /y:800
> 2. Set "Undo" to:   qres.exe /x:3440 /y:1440

Then, under "Detached Commands", I wrote: (please note my computer's user is "natha", which will be switched by your user)
"C:\Users\natha\AppData\Local\Playnite\Playnite.DesktopApp.exe"

Finally, in the Playnite app, I went to settings and clicked on the box titled "launch in full screen mode".

The only thing I can't figure out is how to automatically close the Playnite app after logging off, so any suggestions would be greatly appreciated! Otherwise, I hope you had an easier time than me - Happy gaming!

![1](https://user-images.githubusercontent.com/100257599/210308305-2a5c8942-27aa-4506-8d9c-8886ecd59eac.JPG)

**Comment by OUSooner911** [comment]
Also note that for the Detached Commands, the quotation marks are needed

**Comment by chessset5** [comment]
Btw, if you want the path to go to the current user's folder, use the Environmental Variable or wild card %UserProfile%. 

So the one above would be `"C:\Users\%UserProfile%\AppData\Local\Playnite\Playnite.DesktopApp.exe"` and that would go directly to the current user's Playnite.DesktopApp.exe. No matter what user is logged on.

**Reply by chessset5** [reply]
`"C:\Users\%UserProfile%\AppData\Local\Playnite\Playnite.FullscreenApp.exe"` Also I find this command works best for those who use a controller.

[a comment by an automated account is not shown]
